lib/resty/openssl/x509/init.lua

```diff
@@ -746,7 +751,12 @@ function _M:get_issuer_alt_name()
   got = ffi_cast("GENERAL_NAMES*", got_ref)
   local lib = require("resty.openssl.x509.altname")
   -- the internal ptr is returned, ie we need to copy it
-  return lib.dup(got)
+  local res, err = lib.dup(got)
```

Comment:
it seems okay if we just do

```lua
  local got_ref = got
  got = ffi_cast("GENERAL_NAMES*", got_ref)
  ffi_gc(got, stack_lib.gc_of("GENERAL_NAME"))
```
instead, then later on we don't need to anchor `got`, as `lib.dup` already anchors it.